### PR TITLE
Attempting Discord Integration

### PR DESCRIPTION
**Pull request by PyEddy**
Attempting to add Discord guild integration using Zenora

Still a WIP, some parts work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,9 @@ First you will want to install the following packages by executing the following
 `sudo apt-get update`
 `sudo apt-get install -y python3 python3-pip sqlite3 git unzip libmagickwand-dev`
 
+Then install the Zenora library for python by executing this command
+`pip install Zenora`
```
Comment (aronwk-aaron) [inline README.md:115]: Why?
Add it to the requirements via pip freeze

File: app/main.py [hunk 2]
```diff
@@ -1,27 +1,62 @@
-from flask import render_template, Blueprint, send_from_directory
+from flask import render_template, Blueprint, send_from_directory, request, session, redirect
 from flask_user import current_user, login_required
 
 from app.models import Account, CharacterInfo, ActivityLog
 from app.schemas import AccountSchema, CharacterInfoSchema
+from zenora import APIClient
+from app.settings import TOKEN, CLIENT_SECRET, REDIRECT_URI, OAUTH_URL, DISCORD_INTEGRATION
```
Comment (aronwk-aaron) [inline app/main.py:7]: Don't import from there, handle it down in the setting and add it to the APP_CONFIG

File: README.md (hunk 1 above)
Comment (PyEddy) [reply README.md]: Yep, this is corrected now

Its been added to requirements

File: app/main.py (hunk 2 above)
Comment (PyEddy) [reply app/main.py]: I think I managed to correct this too

I think with Zenora being an external module, I had to use ```with scheduler.app.app_context():``` to get it working when adding app configs to the paramaters of APIClient

File: app/main.py [hunk 3]
```diff
@@ -1,27 +1,63 @@
...
 from app.schemas import AccountSchema, CharacterInfoSchema
+from app import scheduler
+from zenora import APIClient
 
 main_blueprint = Blueprint('main', __name__)
 
 account_schema = AccountSchema()
 char_info_schema = CharacterInfoSchema()
 
+with scheduler.app.app_context():
```
Comment (aronwk-aaron) [inline app/main.py:14]: no, use the current app, not the scheduler

File: app/main.py [hunk 4]
```diff
@@ -1,27 +1,63 @@
...
+    client = APIClient(current_app.config['TOKEN'], client_secret=current_app.config['CLIENT_SECRET'])
```
Comment (aronwk-aaron) [inline app/main.py:15]: this is an extension, register it as such

File: app/main.py (hunk 3 above)
Comment (aronwk-aaron) [reply app/main.py:14]: Actually, see comment below and remove this

**Comment by aronwk-aaron** [comment]
see comment on #36

**Pull request closed without merging**